content/en/docs/helm/helm_install.md

```diff
@@ -50,6 +50,8 @@ set for a key called 'foo', the 'newbar' value would take precedence:
 To check the generated manifests of a release without installing the chart,
 the '--debug' and '--dry-run' flags can be combined.
 
+CRD manifests in the crds/ directory can't be validated by `--dry-run` flag. See [Helm's CRDs caveats](docs/chart_best_practices/custom_resource_definitions.md#some-caveats-and-explanations) for more details.
```

Comment:
CRD manifests in the crds/ directory can't be validated by `--dry-run` flag. See [Helm's CRDs caveats](docs/chart_best_practices/custom_resource_definitions.md#some-caveats-and-explanations) for more details. --> **Note:** CRD manifests in the `crds/` directory can't be validated by `--dry-run` flag. See [Helm's CRDs caveats](docs/chart_best_practices/custom_resource_definitions.md#some-caveats-and-explanations) for more details.

Reply:
@hickeyma great point!
I've notice that a lot of english commands page uses single quotes instead of backticks, as you can see in lines 50 and 51 within this file (helm_install.md).
Would you find useful to change these single quotes to backticks?

Reply:
> Would you find useful to change these single quotes to backticks in order to have a better code highlighting?

It might be worth opening an issue to document this and see what the standard should be to have consistency across the documentation.

Reply:
Sure I'll open it! Thanks for the reviews!